src/components/pages/Dashboard/InstructorStats.js

```diff
@@ -22,16 +22,14 @@ const initialValues = {
   totalCourse: 0,
 };
 function InstructorStats(props) {
-  const { authState, authService } = useOktaAuth();
-  const { idToken } = authState;
+  // const { idToken } = authState;
   const dispatch = useDispatch();
   const [stats, setStats] = useState(initialValues);
 
   useEffect(() => {
     if (!props.user.name) {
       dispatch(getCurrentUser(idToken, authState, authService));
     }
-    // eslint-disable-next-line
   }, [dispatch, idToken]);
```

Comment:
Another one here - lots of errors for undefined values in this file. Let's work on fixing these!